Resources/public/css/layout.css

```diff
@@ -355,6 +355,10 @@ body.fixed .content-header .navbar.stuck {
     border-radius: 0;
 }
 
+.box .nav-stacked > li {
```

Comment:
Added the class `sonata-search-result-list` for the search result. Hope your now fine with it ;)